### PR TITLE
add deployed cypress contracts address

### PR DESCRIPTION
**Pull request by bayram98**
This PR is ready to review and merge.

Changes:
- Add `Cypress` `Prepayment`, and `RR Coordinator` contract address
- Add `cancelAccount` hardhat task
- Fix contract formatting

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -56,10 +56,12 @@ const config: HardhatUserConfig = {
       default: 0
     },
     prepayment: {
-      baobab: '0xf37a736b476fCEaB261371A3B3B330772630b0bF'
+      baobab: '0xf37a736b476fCEaB261371A3B3B330772630b0bF',
+      cypress: '0xCD54643C2Cd905e31d1ca1bF6617cbA9746F2e37'
```
Comment (martinkersner) [inline hardhat.config.ts:60]: @bayram98 Can you please also update addresses for Cypress in README?

File: contracts/VRFConsumer.sol [hunk 2]
```diff
@@ -1,77 +1,60 @@
...
+    }
+
+    function fulfillRandomWords(
+        uint256 /* requestId */,
+        uint256[] memory randomWords
+    ) internal override {
+        // requestId should be checked if it matches the expected request
+        // Generate random value between 1 and 50.
+        sRandomWord = (randomWords[0] % 50) + 1;
+    }
```
Comment (martinkersner) [inline contracts/VRFConsumer.sol:59]: @bayram98 Can you please also add a `withdrawTemporary` to this mock contract? Without that, consumer could not get money back if they request VRF through temporary account is not fulfilled.

File: deploy/0-VRFConsumer.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   console.log('0-VRFConsumer.ts')
 
-  if (network.name != 'baobab') {
+  if (network.name == 'localhost') {
```
Comment (martinkersner) [inline deploy/0-VRFConsumer.ts:11]: I am not sure, why we had this condition, but I think we should remove it and allow to deploy to any network. 🤔

**Review by martinkersner — APPROVED** [review]
LGTM! Just few small suggestions.

**Review by martinkersner — APPROVED** [review]
LGTM!

File: deploy/0-VRFConsumer.ts (hunk 3 above)
Comment (martinkersner) [inline deploy/0-VRFConsumer.ts:11]: We could delete this condition block.

**Comment by bayram98** [comment]
@martinkersner All the conflict resolved. I am merging the PR.

**Pull request merged**